src/test/java/pageObjects/chpldownload_page.java

```diff
@@ -0,0 +1,29 @@
+package pageObjects;
+import static org.testng.Assert.assertTrue;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+
+import stepDefinitions.hooks;
+
+public class chpldownload_page {
+	
+	private WebDriver driver;
+	private static WebElement element = null;
+	
+	public static void print_page_title(WebDriver driver) {
+		driver = hooks.driver;
+		driver.manage().window().maximize();
+		String actualTitle = driver.getTitle();
+		System.out.println("page title is : "+actualTitle);
```

Comment:
We shouldn't have `System.out.println` in _any_ of our real code (maybe; there are never any absolutes, including this one, but still...). It's great for debugging, but a Page Object Class should just return things